FlowCrypt/Functionality/Services/Organisational Rules Service/OrganisationalRulesService.swift

```diff
@@ -14,10 +15,14 @@ protocol OrganisationalRulesServiceType {
     func fetchOrganisationalRules(for email: String) -> Promise<OrganisationalRules>
 
     func getSavedOrganisationalRulesForCurrentUser() -> OrganisationalRules
+
+    func getEmailKeyManagerPrivateKeys() -> Promise<OrganisationalRulesService.GetEKMKeysResult>
```

Comment:
Here it's getting mixed up - `GetEKMKeysResult` and `getEmailKeyManagerPrivateKeys` should not be under `OrganisationalRulesService`. EKM is a separate server, it should have its own service.